web/src/main/java/cz/muni/fi/pa165/team/match/MatchController.java

```diff
@@ -0,0 +1,111 @@
+package cz.muni.fi.pa165.team.match;
+
+import cz.muni.fi.pa165.response.RedirectResponse;
+import cz.muni.fi.pa165.team.Team;
+import cz.muni.fi.pa165.team.TeamPlayer;
+import cz.muni.fi.pa165.team.TeamPlayerRepository;
+import cz.muni.fi.pa165.team.TeamRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.time.format.DateTimeFormatter;
+import java.util.*;
+
+/**
+ * @author Tomas Smid <smid.thomas@gmail.com>
+ */
+@Controller
+public class MatchController
+{
+
+    private final TeamMatchFacade teamMatchFacade;
+
+    private final TeamRepository teamRepository;
+
+    private final TeamMatchGoalRepository teamMatchGoalRepository;
+
+    private final TeamPlayerRepository teamPlayerRepository;
+
+    @Autowired
+    public MatchController(
+        TeamMatchFacade teamMatchFacade,
+        TeamRepository teamRepository,
+        TeamMatchGoalRepository teamMatchGoalRepository,
+        TeamPlayerRepository teamPlayerRepository)
+    {
+        this.teamMatchFacade = teamMatchFacade;
+        this.teamRepository = teamRepository;
+        this.teamMatchGoalRepository = teamMatchGoalRepository;
+        this.teamPlayerRepository = teamPlayerRepository;
+    }
+
+    @RequestMapping(value = "/matches", method = RequestMethod.GET)
+    public ModelAndView viewMatchList()
+    {
+        List<Integer> homeTeamsGoals = new ArrayList<>();
+        List<Integer> awayTeamsGoals = new ArrayList<>();
+        Collection<TeamMatch> playedMatches = teamMatchFacade.findAllPlayedMatches();
+        Collection<TeamMatch> plannedMatches = teamMatchFacade.findAllPlannedMatches();
+        ModelAndView modelAndView = new ModelAndView("team/match/list");
+
+        for (TeamMatch pmatch : playedMatches) {
```

Comment:
vubec na cely tady tohle sbirani dat bych si udělal nějakou metodu ve fasádě, ktera je poskládá do nějaké vhodné struktury, ze které se to bude pěkně vypisovat